### PR TITLE
[bitnami/*] Fix markdown linter issues

### PR DESCRIPTION
**Pull request by carrodher**
Fix markdown linter issues in all markdown files present in this repo

⚠️ Please note those changes need to be replicated in our internal repo

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,14 +34,14 @@ If you set your `user.name` and `user.email` git configs, you can sign your comm
...
+    Author: Joe Smith <joe.smith@example.com>
+    Date:   Thu Feb 2 11:41:15 2018 -0800
 
-    Update README
+        Update README
 
-    Signed-off-by: Joe Smith <joe.smith@example.com>
-```
+        Signed-off-by: Joe Smith <joe.smith@example.com>
+    ```
```
Comment (andresbono) [inline CONTRIBUTING.md:44]: This block of code is not properly rendered

File: bitnami/airflow-scheduler/README.md [hunk 2]
```diff
@@ -61,14 +61,14 @@ If you want to run the application manually instead of using `docker-compose`, t
 1. Create a network
 
   ```console
-  $ docker network create airflow-tier
+  docker network create airflow-tier
   ```
 
-2. Create a volume for PostgreSQL persistence and create a PostgreSQL container
+1. Create a volume for PostgreSQL persistence and create a PostgreSQL container
```
Comment (andresbono) [inline bitnami/airflow-scheduler/README.md:67]: The ordered list is not auto incrementing the index

File: bitnami/zookeeper/README.md [hunk 3]
```diff
@@ -236,11 +236,12 @@ services:
   ...
 ```
 
-### Configuration
+### Zookeeper Configuration
```
Comment (andresbono) [inline bitnami/zookeeper/README.md:239]: The capitalization should be `ZooKeeper`
```suggestion
### ZooKeeper configuration
```

It is wrong in other places in the file though and maybe it is not important to fix it in this PR.

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -12,6 +12,6 @@ If you are subjected to or witness unacceptable behavior, or have any other conc
 
 If you have suggestions to improve this Code of Conduct, please submit an issue or PR.
 
-**Attribution**
+## Attribution
 
-This Code of Conduct is adapted from the Angular project available at this page: https://github.com/angular/code-of-conduct/blob/master/CODE_OF_CONDUCT.md
+This Code of Conduct is adapted from the Angular project available at this page: <https://github.com/angular/code-of-conduct/blob/master/CODE_OF_CONDUCT.md>
```
Comment (andresbono) [inline CODE_OF_CONDUCT.md:17]:
```suggestion
This Code of Conduct is adapted from the Angular project available at this page: <https://github.com/angular/code-of-conduct/blob/main/CODE_OF_CONDUCT.md>
```

File: bitnami/airflow-scheduler/README.md (hunk 2 above)
Comment (carrodher) [reply bitnami/airflow-scheduler/README.md:67]: The default style is `one` according to https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md029---ordered-list-item-prefix

Comment (andresbono) [reply bitnami/airflow-scheduler/README.md:67]: GitHub markdown may not be following that convention, right? See the rendered section: https://github.com/bitnami/containers/blob/e7ea14f56da588fefb95a24693ee396580f7347e/bitnami/airflow-scheduler/README.md#using-the-docker-command-line all the ordered items show the `1.` prefix.

Is it possible to configure the markdown linter to comply with the style required by GitHub?

File: CONTRIBUTING.md (hunk 1 above)
Comment (carrodher) [reply CONTRIBUTING.md]: Now it should be solved

File: bitnami/airflow-scheduler/README.md (hunk 2 above)
Comment (carrodher) [reply bitnami/airflow-scheduler/README.md:67]: I will replace it by ordered lists in the following commit

**Review by dgomezleon — APPROVED** [review]
LGTM

**Review by agomezmoron — APPROVED** [review]

**Pull request merged**